src/main/webapp/college-info.js

```diff
@@ -24,6 +24,18 @@ function init() {
  * Loads the college data from an object passed in local storage.
  */
 function loadSchoolInfo() {
+  const infoDivId = 'info';
+  const reviewsDivId = 'reviews';
+
+  const infoNavButton = document.getElementById(`${infoDivId}-nav`);
+  const reviewsNavButton = document.getElementById(`${reviewsDivId}-nav`);
+
+  infoNavButton.classList.add('active');
+  infoNavButton.addEventListener('click',
```

Comment:
I'd recommend against setting the display in `loadComments` and `loadSchoolInfo` since it's an unexpected side effect from the function names. That's work that's perfect in `init` instead.